### PR TITLE
Improve readme, add ci foundations

### PR DESCRIPTION
**Pull request by inverse**
- Add CI foundations (tests need to be properly implemented)
- Tidy up readme
  - Add security 
  - Tidy Mac -> MacOS 
  - Add pypi badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,3 +64,13 @@ Create a cell in the top of the notebook and paste the content (example below)
...
+
+# Security
+
+For security issues please contact [security@getyourguide.com](mailto:security@getyourguide.com).
+
+# Legal
+
+Copyright 2021 GetYourGuide GmbH.
+
+BambooCoffee is licensed under the Apache License, Version 2.0. See [LICENSE](LICENSE.txt) for the full text.
```
Comment (jeanCarloMachado) [inline README.md]: Db-rocket ?

**Comment by jeanCarloMachado** [comment]
Wow, thanks a lot! Great improvements, feel free to merge.

File: README.md [hunk 2]
```diff
@@ -61,3 +64,13 @@ Create a cell in the top of the notebook and paste the content (example below)
...
 
 Contributions are welcomed!
+
+# Security
+
+For security issues please contact [security@getyourguide.com](mailto:security@getyourguide.com).
+
+# Legal
+
+Copyright 2021 GetYourGuide GmbH.
```
Comment (jeanCarloMachado) [inline README.md]: This line makes it sound like it is closed-source. Wdyt?

File: README.md (hunk 1 above)
Comment (jeanCarloMachado) [reply README.md]: Changed :)

Comment (inverse) [reply README.md]: Ahh sorry - CP error 🗡️

File: README.md (hunk 2 above)
Comment (inverse) [reply README.md]: hmm not sure about this one - We have it on the majority of our other OSS stuff. Perhaps one for the channel to discuss?

Comment (jeanCarloMachado) [reply README.md]: Does not sound like a big deal for GYG (as the project is already on gyg github org). I will just remove this line, if you feel strong about it we can add it back later.

**Pull request merged**